osu.Game/Beatmaps/BeatmapManager.cs

```diff
@@ -163,6 +165,27 @@ public void Download(BeatmapSetInfo beatmapSetInfo, bool noVideo = false)
                     Icon = FontAwesome.fa_superpowers,
                     Text = "You gotta be an osu!supporter to download for now 'yo"
                 });
+
+                if (!beatmapSetInfo.OnlineBeatmapSetID.HasValue)
+                    return;
+
+                // Open download in browser
+                var downloadUrl = $"https://osu.ppy.sh/beatmapsets/{beatmapSetInfo.OnlineBeatmapSetID}";
+
+                // https://brockallen.com/2016/09/24/process-start-for-urls-on-net-core/
+                switch (RuntimeInfo.OS)
```

Comment:
Use host.OpenUrlExternally

Reply:
GameHost cannot be accessed from BeatmapManager @ekrctb 
I'm unsure of how to inject it in BeatmapManager if that's possible

Reply:
`BackgroundDependencyLoader` can be used. Find usage to learn how to use.